docs/AUTHORS.rst

```diff
@@ -15,21 +15,72 @@ This list is sorted by the number of commits per contributor in *descending* ord
      - Contributions
```

Comment:
This file is automatically modified so shouldn't be part of the PR.

Reply:
remove?

Reply:
Yes, please.